packages/core/src/external/ehr/job/create-resource-diff-bundles/steps/contribute/__tests__/ehr-contribute-resource-diff-bundles-direct.test.ts

```diff
@@ -0,0 +1,28 @@
+import { reverseSortIds } from "../ehr-contribute-resource-diff-bundles-direct";
+
+describe("reverseSortIds", () => {
```

Comment:
nit: first level describe is good to be the representation of the file, like "resource-diff-direct" or something.

NABD though, we can edit if/when we add tests for other function from the same file